libs/components/validation/src/lib/modules/url-validation/url-validation-options.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Specifies options for the URL validator component.
+ */
+export interface SkyUrlValidationOptions {
+  /**
+   * Indicates which ruleset version of URL validation to use.
+   *
+   * Ruleset v1 (the default) uses a simple, custom regex.
+   *
+   * Ruleset v2 uses the stricter, 3rd party [validator.js library](https://github.com/validatorjs/validator.js/).
```

Comment:
Since this will be added to the docs, I would make it all one paragraph for readability (with a few minor changes):
```
Indicates the ruleset to use for URL validation. Ruleset 1 uses a regular expression and ruleset 2 uses the third-party [validator.js library](https://github.com/validatorjs/validator.js/).
```

Reply:
resolved in: [update jsdoc for rulesetVersion](https://github.com/blackbaud/skyux/pull/201/commits/6b6eb2ec5b3c2f5d06dcaa4acc524d9de414abac)